drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -83,6 +83,16 @@ def self.authorized?(object, ctx)
     field :phone_numbers, [HmisSchema::ClientContactPoint], null: false
     field :email_addresses, [HmisSchema::ClientContactPoint], null: false
     field :hud_chronic, Boolean, null: true, description: 'Meets the definition for HUD chronically homeless as of today (time of API request)'
+    field :ce_opportunities, [Types::HmisSchema::CeOpportunity], null: false
+
+    def ce_opportunities
+      raise unless Hmis::Ce.configuration.enabled?
+
+      Hmis::Ce::Opportunity.
+        for_client(object).
+        order(:id).
+        limit(50) # FIXME: add pagination. Just limit to top 50 for now
+    end
```

Comment:
We will probably want this to be `field :eligible_ce_opportunities, Types::HmisSchema::CeOpportunity.page_type, null: false` to support the client dashboard eligible opps page. Not sure what it was intended for but I don't think we have another use for resolving opps on client.

Reply:
Sounds good! I addressed that over here https://github.com/greenriver/hmis-warehouse/pull/5262